commit 7734e9fa887293d8262d895d6277bd98ca9bc79e
Author: John Spray <jspray@redhat.com>
Date:   Thu Sep 27 10:37:36 2018 -0400

    common: remove unused 'avail' field from commands
    
    This was introduced for the now-removed ceph-rest-api
    gateway.  It enabled limiting certain commands
    to be CLI-only or rest-only, but in practice almost
    everything just said "cli,rest" here.
    
    Now that ceph-rest-api is gone, let's remove this
    field.
    
    The CLI client code already tolerated the absence
    of this field, so older CLI clients will not mind.
    
    Signed-off-by: John Spray <john.spray@redhat.com>

diff --git a/src/mgr/DaemonServer.cc b/src/mgr/DaemonServer.cc
index 99ed41d7b0c..3f69ff76915 100644
--- a/src/mgr/DaemonServer.cc
+++ b/src/mgr/DaemonServer.cc
@@ -834,7 +834,7 @@ bool DaemonServer::_handle_command(
       ostringstream secname;
       secname << "cmd" << setfill('0') << std::setw(3) << cmdnum;
       dump_cmddesc_to_json(&f, secname.str(), mc.cmdstring, mc.helpstring,
-                           mc.module, mc.req_perms, mc.availability, 0);
+                           mc.module, mc.req_perms, 0);
       cmdnum++;
     };
 
@@ -858,7 +858,7 @@ bool DaemonServer::_handle_command(
 
   bool is_allowed;
   if (!mgr_cmd) {
-    MonCommand py_command = {"", "", "py", "rw", "cli"};
+    MonCommand py_command = {"", "", "py", "rw"};
     is_allowed = _allowed_command(session, py_command.module,
       prefix, cmdctx->cmdmap, param_str_map, &py_command);
   } else {
